packages/es-components/src/components/containers/menu/MenuSection.js

```diff
@@ -24,7 +25,7 @@ const StyledChildrenContainer = styled.section`
 
 const MenuSection = (props, context) => {
```

Comment:
since we aren't using the `context` param anymore, let's get rid of this. could we also change this to `function MenuSection`? the arrow func variables are hard for me to  read at a glance.

Reply:
I thought I did. I must have lost those changes at some point, probably when I was branch hopping. Thanks for catching this.